File: rust/easy/0242_valid_anagram.rs
impl Solution {
    pub fn is_anagram(s: String, t: String) -> bool {
        if s.len() != t.len() {
            return false;
        }
        let mut count = vec![0; 26];
        for c in s.chars() {
            count[(c as u8 - b'a') as usize] += 1;
        }
        for c in t.chars() {
            count[(c as u8 - b'a') as usize] -= 1;
        }
        count.iter().all(|&e| e == 0)
    }
}

File: python/easy/0242_valid_anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        freq = [0 for ch in range(26)]
        for ch in s:
            freq[ord(ch) - ord('a')] += 1
        for ch in t:
            freq[ord(ch) - ord('a')] -= 1
        return all(val == 0 for val in freq)
